x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/alerts_charts.cy.ts

```diff
@@ -103,7 +103,8 @@ describe('KPI visualizations in Alerts Page', { tags: ['@ess', '@serverless'] },
     });
   });
 
-  context('Histogram legend hover actions', () => {
+  // For some reason this suite is failing in CI while I cannot reproduce it locally
```

Comment:
do we need a follow up issue here? 🤔

Reply:
Yup, already logged here: https://github.com/elastic/kibana/issues/195355